src/ypd/templates/ProjectList.j2

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
```

Comment:
So we don't need this file this is the home page, make this what the posting page the user will see for a single listing